tensorflow/core/kernels/mkl_conv_ops.h

```diff
@@ -37,23 +37,16 @@ limitations under the License.
 #include "tensorflow/core/platform/macros.h"
 #include "tensorflow/core/util/padding.h"
 #include "tensorflow/core/util/tensor_format.h"
-
 #include "tensorflow/core/util/mkl_util.h"
```

Comment:
Could you please put this line before `#include "tensorflow/core/util/padding.h"`? The headers in the same group should be ordered lexicographically.

Reply:
Thank you for your suggestion which has been taken with code change.